pyrightconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "include": [
-    "times_reader",
+    "xl2times",
     "utils",
     "times_excel_reader.py"
```

Comment:
Should this one be renamed as well?

Reply:
Ah, we should remove that. It's a remnant of the previous folder structure, before we started using `pyproject.toml` and `setuptools`